efficientdet/keras/efficientdet_arch_keras_test.py

```diff
@@ -28,12 +28,30 @@
 
 class KerasTest(tf.test.TestCase):
 
+  def test_backbone(self):
+    inputs_shape = [1, 512, 512, 3]
+    with tf.Session(graph=tf.Graph()) as sess:
+      feats = tf.ones(inputs_shape)
+      config = hparams_config.get_efficientdet_config('efficientdet-d0')
+      tf.random.set_random_seed(SEED)
+      feats, _ = efficientdet_arch_keras.build_backbone(feats, config)
+      sess.run(tf.global_variables_initializer())
+      feats1 = sess.run(feats)
+    with tf.Session(graph=tf.Graph()) as sess:
+      feats = tf.ones(inputs_shape)
+      config = hparams_config.get_efficientdet_config('efficientdet-d0')
+      tf.random.set_random_seed(SEED)
+      feats = legacy_arch.build_backbone(feats, config)
+      sess.run(tf.global_variables_initializer())
+      feats2 = sess.run(feats)
+    for key in list(feats.keys()):
+      self.assertAllEqual(feats1[key], feats2[key])
+
   def _test_model_output(self):
```

Comment:
Submodules could get same results with tf1.x model.
The difference may be caused by Keras model.
The test case could be removed.
Please verify it.